### ISSUE TITLE
Not working on Raspbian OS 64 bit.

### ISSUE THREAD

**Issue by YiweiMao** [geekworm-com/x728#7]
rc.local file is still modified by this software as expected but the X728 power is cut ~6 s after the button is pressed to turn it on. The blue LED fades away quickly and turns off at the same time power is cut. 

Curiously, the button can still be used to safely shutdown. 

I'm on Raspbian OS 64 bit downloaded here https://www.raspberrypi.com/software/operating-systems/
![image](https://user-images.githubusercontent.com/46410481/154410099-03477ea1-c6e4-4248-8330-52f306d9c54e.png)

**Comment by YiweiMao** [comment]
Update: May have fried the board? I was using GPIO26 as a sensor input. 

The documentation for the GPIO used by the X728 https://wiki.geekworm.com/X728-hardware#Pins_and_GPIO_used
says it does not use GPIO26 so I went ahead and used it assuming it was accurate. 
![image](https://user-images.githubusercontent.com/46410481/154412158-19c6b421-3402-4b4b-b17a-9044aabdc353.png)
It includes a NEW for V2.1 but does not indicate the other changes in V2.0 where
PIN33 (GPIO13) is replaced by the PIN37 (GPIO26) to avoid the I2S interface from v1.3 to v2.0.

Please fix the documentation so no one else makes the mistake.

**Comment by ricardojmveiga** [comment]
I followed this link http://www.suptronics.com/miniPCkits/x728_v2.2-software.html for the X728 v2.0 on Raspbian OS 64 bits Bullseye, and it works flawlessly.

**Comment by keeper18** [comment]
Hi. Can somebody help with RPI4 8GB Raspbian 64bit?I try to run sudo bash x728-v2.1.sh downloaded from 
git clone https://github.com/geekworm-com/x728 and I have message "File not found"

**Comment by rkolbi** [comment]
> sudo bash x728-v2.1.sh

Do an 'ls' to make sure you are in the correct directory and can see the file first, if so - try:
sudo ./x728-v2.1.sh

**Comment by keeper18** [comment]
> > sudo bash x728-v2.1.sh
> 
> Do an 'ls' to make sure you are in the correct directory and can see the file first, if so - try: sudo ./x728-v2.1.sh

Thanks for help.I'm in home/keeper18/ directory due to "ls" command.Now I try another manual from http://www.suptronics.com/miniPCkits/x728_v2.2-software.html similar to first one and I can totally shutdown RPI by  "x728off "  command or external switch but i don't know how to automatically load bat.py ?Can you help rkolbi?

**Comment by rkolbi** [comment]
It looks like section 3.1 on that page tells you how to add it to cron. Since the python scripts outputs text, you cannot simply add it to the end of rc.local.

**Comment by keeper18** [comment]
> It looks like section 3.1 on that page tells you how to add it to cron. Since the python scripts outputs text, you cannot simply add it to the end of rc.local.

Yes I know I downloaded main/bat.py but they don't load when raspbian starts...

**Comment by rkolbi** [comment]
Did you execute the bash script that installs the power button files? Maybe this will help - I have made a simple UPS python script that is ran at boot, can see it here: https://raw.githubusercontent.com/rkolbi/voron2.4/main/Tap-Tap-Sensorless-Huvud/rPi%20Files/X728_UPS.txt

**Comment by YiweiMao** [comment]
I used systemd to enable /etc/rc-local and get the scripts running on startup. Here's my documentation on my install process
https://openhsi.github.io/openhsi/tutorial_installing_linux.html#4.5-Installing-the-Battery-Hat-software

**Comment by keeper18** [comment]
> Did you execute the bash script that installs the power button files? Maybe this will help - I have made a simple UPS python script that is ran at boot, can see it here: https://raw.githubusercontent.com/rkolbi/voron2.4/main/Tap-Tap-Sensorless-Huvud/rPi%20Files/X728_UPS.txt

Yes.All script works except buzzer because v2.0 have no buzzer.I can reboot hold 2s, shut down press 4s and force shutdown pressing long...but till reboot raspbian.How to load script showing voltage/charge automatically because there is no bar icon and battery level leds are in the box?
